software-setup/rstudio-login.md

```diff
@@ -51,28 +51,28 @@ If you forget your password at any time, Slack one of the Data Lab team members
 
 You can stop your current R session with the orange circular on/off button in the upper right corner:
 
-<img src = "screenshots/rstudio-session-buttons.png" width = "300">
+<img src = "screenshots/rstudio-session-buttons.png" alt = "Stop/restart current R session in RStudio Server"  width = "300">
 <br>
 <br>
```

Comment:
Noting `<br>` tags here, maybe we want to remove them? Depends how it looks rendered.

Reply:
I like them in _this section_ in particular where we have several back-to-back screenshots. The newlines, imo, help space things out to be a little less cluttered.

With the 2 `<br>`s, I feel like I get a better sense of which text goes with which image:

<img width="583" alt="Screenshot 2023-09-26 at 4 05 25 PM" src="https://github.com/AlexsLemonade/training-specific-template/assets/4701111/5afe3a33-c7b5-4bd4-8752-eec1967aab7b">